packages/angular/src/generators/application/application.ts

```diff
@@ -107,6 +108,18 @@ export async function applicationGenerator(
     });
   }
 
+  if (options.mfe) {
+    if (!options.mfeType) {
```

Comment:
I'd either make it an error or default to say "remote".

Reply:
Interesting, I have the `setup-mfe` generator default to a `shell` app. I wonder if I should also change that to default to `remote`